packages/devel/binutils-or1k/package.mk

```diff
@@ -48,6 +48,7 @@ make_host() {
 
 makeinstall_host() {
   cp -v ../include/libiberty.h ${SYSROOT_PREFIX}/usr/include
+  make -C libsframe install # bfd is reliant on libsframe
```

Comment:
Is bfd necessary for or1k? It is technically correct - not sure why the buildhost for Allwinner builds is not having a problem. Should the bfd line go? @jernejsk

Reply:
Not really sure if it's necessary, but this at least fixes the toolchain build for me

Reply:
I’ll give mold/gold/bfd a try on a build. I’m assuming you are doing `PROJECT=Allwinner ARCH=aarch64 DEVICE=H6 make image`

Reply:
I didn’t reproduce an error with 

`PROJECT=Allwinner ARCH=aarch64 AUTOREMOVE=yes DEVICE=H6 make image`

```
LibreELEC.kernel11$ ls -latr target/
total 1404912
drwxr-xr-x  3 docker docker      4096 Dec  2 04:37 addons
drwxr-xr-x 39 docker docker      4096 Jan  5 02:29 ..
-rw-r--r--  1 docker docker  23552008 Jan  5 02:31 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194.kernel
-rw-r--r--  1 docker docker 129667072 Jan  5 02:31 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194.system
-rw-r--r--  1 docker docker 154112000 Jan  5 02:31 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194.tar
-rw-r--r--  1 docker docker       125 Jan  5 02:31 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194.tar.sha256
-rw-r--r--  1 docker docker 141391534 Jan  5 02:32 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-beelink-gs1.img.gz
-rw-r--r--  1 docker docker       140 Jan  5 02:32 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-beelink-gs1.img.gz.sha256
-rw-r--r--  1 docker docker 141414759 Jan  5 02:32 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-orangepi-3.img.gz
-rw-r--r--  1 docker docker       139 Jan  5 02:32 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-orangepi-3.img.gz.sha256
-rw-r--r--  1 docker docker 141414507 Jan  5 02:32 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-orangepi-3-lts.img.gz
-rw-r--r--  1 docker docker       143 Jan  5 02:32 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-orangepi-3-lts.img.gz.sha256
-rw-r--r--  1 docker docker 141389745 Jan  5 02:33 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-orangepi-lite2.img.gz
-rw-r--r--  1 docker docker       143 Jan  5 02:33 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-orangepi-lite2.img.gz.sha256
-rw-r--r--  1 docker docker 141389464 Jan  5 02:33 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-orangepi-one-plus.img.gz
-rw-r--r--  1 docker docker       146 Jan  5 02:33 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-orangepi-one-plus.img.gz.sha256
-rw-r--r--  1 docker docker 141427028 Jan  5 02:33 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-pine-h64.img.gz
-rw-r--r--  1 docker docker       137 Jan  5 02:33 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-pine-h64.img.gz.sha256
-rw-r--r--  1 docker docker 141427230 Jan  5 02:34 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-pine-h64-model-b.img.gz
-rw-r--r--  1 docker docker       145 Jan  5 02:34 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-pine-h64-model-b.img.gz.sha256
-rw-r--r--  1 docker docker 141388914 Jan  5 02:34 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-tanix-tx6.img.gz
-rw-r--r--  1 docker docker       138 Jan  5 02:34 LibreELEC-H6.aarch64-12.0-devel-20240105022924-f87e194-tanix-tx6.img.gz.sha256
drwxr-xr-x  3 docker docker     98304 Jan  5 02:34 .


docker@24ee90e044ed:/var/media/DATA/home-rudi/LibreELEC.kernel11$ git diff
diff --git a/distributions/LibreELEC/options b/distributions/LibreELEC/options
index 5a663f4f05..b8323fe019 100644
--- a/distributions/LibreELEC/options
+++ b/distributions/LibreELEC/options
@@ -36,7 +36,7 @@
   MOLD_SUPPORT="no"
 
 # default linker (bfd / gold / mold)
-  DEFAULT_LINKER="gold"
+  DEFAULT_LINKER="mold"
 
 # HARDENING (security relevant linker and compiler flags) support
   HARDENING_SUPPORT="no"
```

Reply:
Build seems to be fine without this commit (it's 10 months old ;), so dunno what fixed it, but I'll drop it then

Reply:
Okay, so there still is a problem, apparently I just got lucky yesterday with the build order.
But that looks like general binutils problem for all toolchains then, not just or1k